framework/doc/content/source/meshgenerators/AddMetaDataGenerator.md

```diff
@@ -0,0 +1,15 @@
+# AddMetaDataGenerator
+
+!syntax description /Mesh/AddMetaDataGenerator
+
+## Overview
+
+This MeshGenerator object allows the user to add extraneous Mesh Metadata to the input mesh. This would allow the input mesh, especially from `FileMeshGenerator`, to be compatible with those mesh generators (e.g., mesh generators in the Reactor Module) that depend on Mesh Metadata to function.
+
+## Further AddMetaDataGenerator Documentation
```

Comment:
Probably should be an example of starting from a clean exodus

what do we have that uses metadata?
- drums
- trimmability
- pitches and sizes for making patterns
is there anything outside the reactor module?

Reply:
It seems that only Reactor Module is using meta data. I can add a test case to one of them it that works for you.